eng/common/scripts/Test-SampleMetadata.ps1

```diff
@@ -358,6 +358,7 @@ begin {
         "ef-core",
         "ef6",
         "expression-studio",
+        "language-service",
```

Comment:
While in here can you update the comment link to https://review.docs.microsoft.com/en-us/help/contribute/metadata-taxonomies?branch=main#product (i.e. update master to main).

Reply:
Ah, man - and the subordinate pipelines were just about done! But, yeah. 😁